src/core/TChem_Driver.cpp

```diff
@@ -163,9 +163,9 @@ int TChem_getSpeciesName(int * index, char* result, const std::string::size_type
 
 std::string TChem::Driver::getSpeciesName(int *index){
   const auto speciesNamesHost = Kokkos::create_mirror_view(_kmcd_host.speciesNames);
-  Kokkos::deep_copy(speciesNamesHost, _kmcd_host.speciesNames);
+//  Kokkos::deep_copy(speciesNamesHost, _kmcd_host.speciesNames);
```

Comment:
Should this comment be deleted?

Reply:
This comment is outdated and addressed back in 15184967c71d9d00e7121cff2ad3647a3e2a5a48